### PR TITLE
Updates to Arabic locale before OJS3.1

### PR DESCRIPTION
**Pull request by vormia**
Some minor updates to Arabic language files.

### CODE REVIEW

File: plugins/pubIds/doi/locale/ar_IQ/locale.xml [hunk 1]
```diff
@@ -39,16 +39,18 @@
...
-	<message key="plugins.pubIds.doi.manager.settings.doiReassign.confirm">هل أنت متأكد من رغبتك في إزالة كل ما موجود من DOI؟</message>	
+	<message key="plugins.pubIds.doi.manager.settings.doiReassign.confirm">هل أنت متأكد من رغبتك في إزالة كل ما موجود من DOI؟</message>
 
+	<message key="plugins.pubIds.doi.manager.settings.doiAssignJournalWide">تعيين معرفات DOI</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiAssignJournalWide.description">قم بتعيين معرفات DOI لجميع المكونات المنشورة في المجلة والتي لم يسبق لها أن حصلت على تلك المعرفات. هذا الإجراء لا يمكن استعماله عند تهيئة اللاحقة بشكل منفرد. إن قمت بتعديل إعدادات DOI أعلاه، لطفاً قم بحفظ التغييرات قبل الشروع بهذا الإجراء. تعيين معرفات DOI قد يستغرق وقتاً طويلاً، إعتماداً على عدد المكونات المنشورة في المجلة.</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiAssignJournalWide.confirm">Are you sure you wish to assign DOIs to all published objects that have not been assigned DOIs?</message>
```
Comment (mtub) [inline plugins/pubIds/doi/locale/ar_IQ/locale.xml]: This is still English...

**Comment by vormia** [comment]
Thanks @mtub for the call.
Line 53 has been translated.

File: plugins/pubIds/doi/locale/ar_IQ/locale.xml (hunk 1 above)
Comment (vormia) [inline plugins/pubIds/doi/locale/ar_IQ/locale.xml]: The line had been updated by the Arabic script.

**Comment by mtub** [comment]
Thanks, @vormia !

**Pull request merged**